5.3Parseltongue.py

```diff
@@ -0,0 +1,20 @@
+def find_secrets(file_name):
+    """
+    find the secret messages in the resources/logo.jpg file
+    :param file_name: the resources/logo.jpg file
+    :return: generator for the secret messages
+    """
+    with open(file_name, 'rb') as file:
```

Comment:
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names. 'file' is not good.